.github/workflows/production.yml

```diff
@@ -151,3 +151,17 @@ jobs:
           max_duration: "21600" # Run maximum 6 hours
           command: "{/bin/sh}"
           args: "{-c,python manage.py ingest_data ontology_tree; python manage.py ingest_data ontology_word}"
+
+      - name: Harvest administrative division indexes
+        uses: City-of-Helsinki/setup-cronjob-action@main
+        with:
+          name: harvest-administrative-division-indexes
+          image_repository: ghcr.io/city-of-helsinki/${{ github.event.repository.name }}/sources
+          image_tag:  ${{ github.sha }}
+          secret_name: project-staging-sources-secret
+          kubeconfig_raw: ${{ env.KUBECONFIG_RAW}}
+          target_namespace: ${{ secrets.K8S_NAMESPACE_STAGING }}
```

Comment:
"staging"

Reply:
This and the above one were caused by a copy-paste mistake, good catch! 💯